### ISSUE TITLE
Создать страницу об Anki и интервальном повторении

### ISSUE THREAD

**Issue by Freika** [codenamecrud/codenamecrud#58]

**Comment by sdilshod** [comment]
Anki - это  https://ru.wikipedia.org/wiki/Anki ?
Можно коротко описать задачу? @Freika

**Comment by Freika** [comment]
Да, эта программа. Нужно рассказать об интервальном запоминании и почему это гораздо надежнее и лучше, чем "прочитал и забыл".

**Comment by sdilshod** [comment]
То есть нужна создать статическую страницу где описывается об программе Anki и интервалном повторение? @Freika

**Comment by Freika** [comment]
Именно так. А потом мы добавим в верхнее меню еще один дропдаун, куда поместим ссылки на эту статью и на статью об английском языке.
